### ISSUE TITLE
different handling of statics

### ISSUE THREAD

**Issue by cataclysmic** [tobiasraabe/otree_virtual_machine_manager#59]
Instead of writing a replacement handler for each new file could we iterate over the statics folder structure and replace ALL standardized the replacements?

**Comment by tobiasraabe** [comment]
👍 Should it be included in the forthcoming release?

**Comment by cataclysmic** [comment]
Yes. I think it would allow for more flexibility. One could write another statics command and simply throw it in the statics folder in the correct location and the necessary bits and pieces that would require changes specific to each user would be replaced on initialization.

**Comment by tobiasraabe** [comment]
Ok then, we have to open issues for the forthcoming release (I'm sorry, I had to play around with milestones :)). Pick one and I will choose the other one.
